distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java

```diff
@@ -166,11 +172,33 @@ public void initFacades() {
      */
     @org.ops4j.pax.exam.Configuration
     public Option[] config() throws URISyntaxException {
+        basePort = findPortNumber(20000);
+
+        HTTP_PORT = String.valueOf(basePort + 1);
```

Comment:
Instead of keeping/modifying all these constant strings, how about using a `final static Map` with these constants changed to keys? I think you could end up managing fewer moving pieces that way. Maybe.